scripts/build_images.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Needed this on macOS to load the newest version of bash, installed by homebrew. Otherwise `declare -A` is an error

```suggestion
#!/usr/bin/env bash
```

Reply:
done